schema/spacewalk/common/views/endpointCatalog.sql

```diff
@@ -0,0 +1,15 @@
+--
+-- Copyright (c) 2025 SUSE LLC
+--
+-- This software is licensed to you under the GNU General Public License,
+-- version 2 (GPLv2). There is NO WARRANTY for this software, express or
+-- implied, including the implied warranties of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+-- along with this software; if not, see
+-- http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+--
+
+CREATE VIEW access.endpointCatalog AS
+SELECT n.namespace, n.access_mode, e.endpoint, e.http_method, e.scope
+FROM access.endpointNamespace en, access.endpoint e, access.namespace n
+WHERE en.namespace_id = n.id AND en.endpoint_id = e.id;
```

Comment:
This is a cross join, isn't it? Is that intended? Seems to me this can also be set as an inner join:

`FROM access.endpoint e JOIN access.endpointnamespace en ON e.id = en.endpoint_id JOIN access.namespace n ON en.namespace_id = n.id;`